contexts/citizenship-v1.jsonld

```diff
@@ -0,0 +1,23 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "ctzn": "https://w3id.org/citizenship#",
+    "schema": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+    "Person": "ctzn:Person",
+
+    "birthCountry": "schema:birthCountry",
```

Comment:
All of this should probably be type scoped like we do in the base VC vocab: https://github.com/w3c/vc-data-model/blob/gh-pages/contexts/credentials/v1#L9

On the fence about that one -- feels like we'll need to type scope based on every type of card, @dlongley thoughts?

Reply:
We should type scope everything to the specific credential/card type.

Reply:
We should also be using `@protected`.

Reply:
Let's do this now so we don't have to do it later.